twine/commands/upload.py

```diff
@@ -60,6 +60,12 @@ def upload(upload_settings: settings.Settings, dists: List[str]) -> None:
     repository_url = cast(str, upload_settings.repository_config["repository"])
 
     print(f"Uploading distributions to {repository_url}")
+    if upload_settings.verbose:
+        for dist_to_upload in uploads:
+            # If the file size is <.1 MB we display in KB
+            file_size, size_unit = utils.get_file_size(dist_to_upload)
+            print(f"   {dist_to_upload} ({file_size:.1f} {size_unit})")
```

Comment:
For consistency with the existing loop through `uploads`, let's use `filename` for the variable.

I think the comment would be better as part of the docstring for `get_file_size`.

Also, nitpick: I think two spaces is sufficient for indentation (and follows the `skip_message` indentation later)

```suggestion
        for filename in uploads:
            file_size, size_unit = utils.get_file_size(filename)
            print(f"  {dist_to_upload} ({file_size:.1f} {size_unit})")
```